### DISCUSSION TITLE
How change priority for comments in the right side?

### DISCUSSION THREAD

**Discussion by KirovAI** [NvChad/NvChad#1392] [Q&A]
When you write code not clearly comments adviced whats wrong, but priority very bad. In first position always displayed "no matters things" as excess probel then warnings and only then errors.
In advance, thanks

**Comment by siduck** [comment]
i dont get this :thinking:

**Comment by KirovAI** [comment]
some like this:
![cursor_line_config](https://user-images.githubusercontent.com/105173528/181466874-3f6c2771-73e7-44b8-ac6a-5c2b58118ad1.png)

**Reply by siduck** [reply]
idk,ask it on neovim sub. I can help only with nvchad stuff, not general neovim stuff as I am a noob myself
